File: usth/ICT2.2/labwork/3/Java/NameCardTest.java
class NameCardTest
{
  public static void main(String... args)
  {
    NameCard card = new NameCard("Foobar Baz", "420-69", "foo@bar.baz");
    System.out.printf("Name: %s\nPhone: %s\nEmail: %s\n",
                      card.getName(), card.getPhone(), card.getEmail());
  }
}

File: usth/ICT2.2/labwork/3/C++/name-card-test.cc
#include <iostream>

#include "name-card.h"

using namespace std;

int
main ()
{
  NameCard card ("Foobar Baz", "420-69", "foo@bar.baz");
  cout << "Name: " << card.get_name () << endl
       << "Phone: " << card.get_phone () << endl
       << "Email: " << card.get_email () << endl;
}
